examples/ArduinoGuitarHero/ArduinoGuitarHero.ino

```diff
@@ -36,11 +34,11 @@ dataForController_t getControllerData(void)
   controllerData.startOn = guitar.BPlus == 1;
 
   //Use analog stick bindings for the controller joystick
-  controllerData.leftStickX = guitar.sX;
-  controllerData.leftStickY = guitar.sY;
-
-  controllerData.l1On = guitar.WB > 2;
+  controllerData.leftStickX = map(guitar.sX, 0, 63, 0, 255); // 0 -> 63
+  controllerData.leftStickY = map(guitar.sY, 0, 63, 0, 255);
 
+  controllerData.rightStickX = map(guitar.WB, -16, 16, 0, 255); // -16 -> 16 (about 12 on a real guitar, 16 is absolute maximum guitar controller can report)
+  //Serial.println(controllerData.leftStickX);

```

Comment:
nitpick: remove commented out debugging line

Reply:
Hey I have modified the code for your nitpick, let me know when you will merge